src/test/java/org/junit/tests/experimental/theories/runner/UnsuccessfulWithDataPointFields.java

```diff
@@ -121,7 +121,7 @@ public void numbers(int x) {
     public void dataPointFieldsMustBeStatic() {
         assertThat(
                 testResult(DataPointFieldsMustBeStatic.class),
-                CoreMatchers.<PrintableResult>both(failureCountIs(2))
+                CoreMatchers.<PrintableResult>both(failureCountIs(1))
```

Comment:
@stefanbirkner only at line 123 or do you mean all of the tests asserting on result.failureCount?